doc/src/modules/physics/mechanics/examples/multi_degree_freedom_holonomic_system.rst

```diff
@@ -0,0 +1,86 @@
+=========================================
+Multi Degree of Freedom Holonomic System
+=========================================
+
+In this example we demonstrate the use of the functionality provided in
+:mod:`sympy.physics.mechanics` for deriving the equations of motion (EOM) of a holonomic
+system that includes both particles and rigid bodies with contributing forces and torques,
+some of which are specified forces and torques. The system is shown below:
+
+.. image:: multidof-holonomic.*
+   :align: center
+
+The system will be modeled using ``JointsMethod``. First we need to create the
+``dynamicsymbols`` needed to describe the system as shown in the above diagram.
+In this case, the generalized coordinates :math:`q_1` represent lateral distance of block from wall,
+:math:`q_2` represents ngle of the compound pendulum from vertical, :math:`q_3`  represents angle of the simple
+pendulum from the compound pendulum. The generalized speeds :math:`u_1` represents lateral speed of block,
+:math:`u_2` represents lateral speed of compound pendulum and :math:`u_3` represents angular speed of C relative to B.
+
+We also create some ``symbols`` to represent the length and
+mass of the pendulum, as well as gravity and others. ::
+
+    >>> from sympy import zeros, symbols
+    >>> from sympy.physics.mechanics import *
```

Comment:
Always do explicit imports in documentation. `from x import y, z, w`

Reply:
All the examples were made like this only. I'll correct it anyway.